packages/renderer/src/lib/container/container-utils.ts

```diff
@@ -181,13 +183,20 @@ export class ContainerUtils {
             name: group.name,
             type: group.type,
             id: group.id,
+            status: group.status,
             engineId: group.engineId,
             containers: [],
           });
         }
         groups.get(group.name).containers.push(containerInfo);
       }
     });
+
+    for (const group of groups.values()) {
+      if (group.type === ContainerGroupInfoTypeUI.COMPOSE)
+        group.status = group.containers.every(c => c.state === 'RUNNING') ? 'RUNNING' : 'STOPPED';
+    }
```

Comment:
maybe something more in line with javascript iteration:
```suggestion
groups.filter(group => group.type === ContainerGroupInfoTypeUI.COMPOSE).forEach(group => group.status = group.containers.every(c => c.state === 'RUNNING') ? 'RUNNING' : 'STOPPED');
```

Reply:
groups is a Map, so there is no filter method. I tried to find something simpler with values() or forEach() but both became more awkward than what's there. Any other ideas?

Reply:
Array.from(map.values()).filter

Reply:
That just looks less efficient to old-school me :), but numbers are low enough I know it wouldn't matter either way. It is more consistent, I've pushed the change.

Reply:
I think the pattern is more the iterable/stream/filterable. It's typescript so it could be transformed to something more efficient maybe but here we read something only with a single pipe


you can also keep your pattern, it was just a suggestion.